modules/kafka/kafka-dest-worker-internal.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2020 Balabit
+ * Copyright (c) 2020 Balazs Scheidler
+ * Copyright (c) 2020 Vivin Peris
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 as published
+ * by the Free Software Foundation, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * As an additional exemption you are allowed to compile & link against the
+ * OpenSSL libraries as published by the OpenSSL project. See the file
+ * COPYING for details.
+ *
+ */
+
+#ifndef KAFKA_TEST_INTERNAL_H_INCLUDED
+#define KAFKA_TEST_INTERNAL_H_INCLUDED
+
+#include "logthrdest/logthrdestdrv.h"
+#include <librdkafka/rdkafka.h>
+#include "kafka-dest-worker.h"
+
+
+
+#define STRING_250_LEN "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" \
+                       "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" \
+                       "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" \
+                       "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" \
+                       "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+
+#define STRING_249_LEN "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" \
+                       "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" \
+                       "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" \
+                       "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" \
+                       "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+
+gchar *_resolve_template_topic_name(KafkaDestWorker *self, LogMessage *msg);
+rd_kafka_topic_t *_calculate_topic_from_template(KafkaDestWorker *self, LogMessage *msg);
+rd_kafka_topic_t *_get_literal_topic(KafkaDestWorker *self);
+rd_kafka_topic_t *_calculate_topic(KafkaDestWorker *self, LogMessage *msg);
+
+#endif
```

Comment:
New line is missing.

Reply:
This is not fixed, don't mark it resolved.

Reply:
Why new line though? I will update it

Reply:
Try calling cat on the file with and without newline.

You might not agree with the review comments some are surely arguable, but not fixing them, neither writing your point, and still marking them as resolved is not cool...

Reply:
Sorry, I missed this. Realized while reading the email notification. Will, go through the review comments once more to make sure I haven't missed anything.

Reply:
I hope its fine now?